### PR TITLE
Support for ADIMLab Gantry

### PR DESCRIPTION
**Pull request by ETE-Design**
**Description**

Right now there is no official support for the ADIMLab Gantry, so people have to make the files themself.

**Benefits**

Add support for ADIMLab Gantry with standard configuration.

### CODE REVIEW

File: config/examples/ADIMLab/Granty/Configuration.h [hunk 1]
```diff
@@ -0,0 +1,2212 @@
...
+ * :[2400, 9600, 19200, 38400, 57600, 115200, 250000, 500000, 1000000]
+ */
+#define BAUDRATE 250000
+
+// Enable the Bluetooth serial interface on AT90USB devices
+//#define BLUETOOTH
+
+// Choose the name from boards.h that matches your setup
+#ifndef MOTHERBOARD
+  #define MOTHERBOARD BOARD_HJC2560C_REV2
```
Comment (thisiskeithb) [inline config/examples/ADIMLab/Granty/Configuration.h:134]: This board doesn't exist in Marlin.

Edit: It looks like TH3D's fork has it: [pins_HJC2560C_REV2.h](https://github.com/houseofbugs/TH3D-Unified-U1.R2/blob/master/TH3DUF_R2/pins_HJC2560C_REV2.h)

Comment (ETE-Design) [reply config/examples/ADIMLab/Granty/Configuration.h:134]: Have just added the missing parts.

Comment (thisiskeithb) [reply config/examples/ADIMLab/Granty/Configuration.h:134]: Don't forget to update [boards.h](https://github.com/MarlinFirmware/Marlin/blob/bugfix-2.0.x/Marlin/src/core/boards.h) & the [Makefile](https://github.com/MarlinFirmware/Marlin/blob/bugfix-2.0.x/Marlin/Makefile) as well.

Comment (ETE-Design) [reply config/examples/ADIMLab/Granty/Configuration.h:134]: Think I'm there now.

**Pull request merged**